Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -464,11 +464,15 @@ public bool IsInLocalNetwork(IPObject address)
         /// <inheritdoc/>
         public bool IsInLocalNetwork(string address)
         {
-            if (IPHost.TryParse(address, out IPHost ep))
```

Comment:
Root cause is the assumption I've made that IPv6 addresses will be enclosed in [ ] like they are in the url - probably as 
eg. fd23:184f:2029:0::/56 will be [fd23:184f:2029:0::/56].

Note to self: IPHost.TryParse needs to be fixed.

Although this will work - the string version of IsInLocalNetwork was intended to permit host name functionality.

I would go with the above change so that IPAddress version of IsInLocalNetwork - and remove this change. I'll submit a separate PR to fix the issue in IPHost.TryParse.